apps/webstore/src/app/pages/auth/login/login.component.html

```diff
@@ -1,5 +1,11 @@
 <main class="flex-container-login">
-    <form *ngIf="!isLoggedIn" (ngSubmit)="onSubmit()" name="loginForm" class="login-form" [formGroup]="loginForm">
+    <form
+        *ngIf="!isLoggedIn"
+        (ngSubmit)="onSubmit(); eventsService.create('webstore.login.clicked.login')"
```

Comment:
Would it not be better to have this call within the functions themselves? so we avoid having too much inline functionality in the html. Just a thought.

Reply:
@Teodor25 it has crossed my mind. Size-wise, it will be the same or even bigger since we still need to store info about which app it came from. We can have it take less space but we save a few bytes in exchange for a less readable entry